### ISSUE TITLE
This package is a joke

### ISSUE THREAD

**Issue by dandv** [jezen/is-thirteen#604]
It should at least do [something](https://github.com/joaojeronimo/rimrafall).

**Comment by idoo** [comment]
there is no joke, I use it every day to compare numbers, please close this PR

**Comment by dandv** [comment]
@idoo: are you being a bit overzealous perhaps? It should only be used every 13th day.

**Comment by ColdGrub1384** [comment]
This library is very useful, I use it in my projects. It's awesome to compare numbers so easily.

**Comment by OtacilioN** [comment]
Really?

**Comment by DUFC-Liam** [comment]
Yeah without this repository I would not be able to check for the number thirteen.
I would like this issue to be closed as it would devastate the coding community as a whole if this project was to be removed.

**Comment by SRGOM** [comment]
@RootBotTeam are you out of your mind? I don't care about the emotions of snowflakes- there are widely used libraries used in production software that depend on this library. Remove this and the internet will come to a halt.

**Comment by sr229** [comment]
You guys never fail to amaze me.
